example/js/scalableTable.js

```diff
@@ -0,0 +1,840 @@
+angular.module('scalable-table', [])
```

Comment:
@trean why don't you use main scalableTable.js directive instead of duplicating it here?